pkg/cmd/browse/browse.go

```diff
@@ -107,12 +107,12 @@ func NewCmdBrowse(f *cmdutil.Factory, runF func(*BrowseOptions) error) *cobra.Co
 func runBrowse(opts *BrowseOptions) error {
 	baseRepo, err := opts.BaseRepo()
 	if err != nil {
-		return fmt.Errorf("unable to determine base repository: %w\nUse 'gh browse --help' for more information about browse\n", err)
+		return fmt.Errorf("unable to determine base repository: %w\n", err)
```

Comment:
Nit: trailing newline character remains in the error message. Error messages should ideally be plain, with as little formatting or punctuation in them, so they can be easily wrapped in each other (like we're doing here using `%w`)